src/styles/main.scss

```diff
@@ -0,0 +1,65 @@
+@import './utils/variables';
+
+body {
+  margin: 0;
+  padding-top: 25px;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: $col-count * $cell-width + $gap * $col-count;
+  margin: 0 auto;
+
+  &__day {
+    display: flex;
+    font-family: Arial, Helvetica, sans-serif;
+    justify-content: center;
+    align-items: center;
+    font-size: 30px;
+    line-height: 100%;
+    box-sizing: border-box;
+    background-color: #eee;
+    width: $cell-width;
+    height: $cell-height;
+    border: 1px solid black;
+
+    &:first-child {
+      margin-left: ($col-count - 1) * $cell-width + $gap * ($col-count - 1);
+    }
+
+    @each $day, $number in $days-of-week {
+      @if $day == $current-day {
+        &:first-child {
+          margin-left: $number * $cell-width + $gap * $number;
```

Comment:
Ensure that `$days-of-week` and `$current-day` are defined and correctly used. This logic is intended to adjust the margin based on the current day, and it must be accurate for the calendar to display correctly.